### PR TITLE
[CFDP-1318] Update Connect Feed

### PR DESCRIPTION
**Pull request by dzwood**
## New Features
We wanted to clean/fix the way we are rendering the Connect feed(`/connect`) on the web. This PR
* cleans up the `ActionBarFeature`
* updates how Groups are pulled in a `HoriztonalCardListFeature`
* adds `PrayerCard` for the displaying a user's prayers in a `HoriztonalCardListFeature`

## Bug Fixes/Updates
* Fixed the `ThemeProvider` bug for the `ActionBar` by using the original `ThemeProvider` from `styled-components`.
* Fixed href links inside of `ActionBar`.
* Updated profile picture inside of Navbar to redirect to `/connect`.

## Known Bugs/Trade-Offs
* Since the `HoriztonalCardListFeature` does not pull in all the needed data for Group cards, I have the `HoriztonalCardListFeature` render a `GroupsProvider` instead when loading Group cards. This is just a _temporary fix_ as we continue refine the process in how we load cards.
* The `HoriztonalCardListFeature`also does not pull in all the props for a Prayer card(_eg. date_), we're skipping them for now. BUT I did create a new ui card(`PrayerCard`) to display the cards more similar to the app.
* The `AvatarListFeature` only redirects to `/connect`, but this is the only place it is being used. So we'll leave it for now.
* GroupsCards links are breaking, need to look into it more.

## Demo/Screenshots
**_To demo:_** Sign in and then click on profile photo or go to `/connect`.
image

## Jira Tickets
[CFDP-1318] Finish the Profile on Web 2.0 to match the functionality of the connect tab on the app

[CFDP-1318]: https://christfellowshipchurch.atlassian.net/browse/CFDP-1318

### CODE REVIEW

File: components/ActionBarFeature/ActionBarFeature.js [hunk 1]
```diff
@@ -41,18 +39,24 @@ const ActionBarFeature = props => {
...
-                label={title}
-                onPressActionItem={e =>
-                  onPressActionItem(e, { action, relatedNode })
-                }
-              />
-              // </ThemeProvider>
+              /**
+               * ! Our ThemeProvider component from /ui-kit is not working properly so we are using the orignal component from the 'style-components' package.
+               */
+              <SCThemeProvider key={i} theme={theme}>
```
Comment (rkdavidson) [inline components/ActionBarFeature/ActionBarFeature.js:45]: Hey @jordanwade or @drewbarontini , any off-hand ideas why _our_ app's `ThemeProvider` wouldn't work here versus the underlying `SCThemeProvider`?

I'm fuzzy on the recent work around theming but the general pattern seems to be overriding the theme in different branches of the component tree.

File: components/HorizontalCardListFeature/HorizontalCardListFeature.js [hunk 2]
```diff
@@ -27,25 +28,51 @@ function HorizontalCardListFeature(props = {}) {
...
 
-  return (
-    <CardCarousel cardsDisplayed={cardsDisplayed} hideArrows={cards.length < 2}>
-      {cards.map((card, i) => {
-        return (
-          <CustomLink
-            as="a"
+  /**
+   * todo : TEMP SOLUTION - In the future we'll want to update how we pull in Group and Prayer cards. Right now for Groups we switch to GroupsProvider to pull in the required fields for GroupCard correctly, bypassing the CardCarousel. For Prayer we'll need to find a way to pull in the correct fields as well.
+   */
```
Comment (rkdavidson) [inline components/HorizontalCardListFeature/HorizontalCardListFeature.js:33]: A temporary solution is infinitely better than no solution! 👏

File: ui-kit/PrayerCard/PrayerCard.stories.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+const cardProps = {
+  date: 'Aug 23, 2020',
+  description: 'This is where my prayer would be displayed.',
+  coverImage: 'https://source.unsplash.com/random/1000x1000',
+  mx: 's',
+};
+
+export const Default = () => (
+  <CardCarousel cardsDisplayed={4} hideArrows>
```
Comment (rkdavidson) [inline ui-kit/PrayerCard/PrayerCard.stories.js:20]: I appreciate that you make the effort to keep up with the Storybook stories — good habit that I should be working on, too.

**Review by rkdavidson — APPROVED** [review]
Looks good to me, tested locally and everything seemed to work as intended.
Nice work on this, @dzwood .

**Review by asleepingpanda — APPROVED** [review]

**Pull request merged**